workos/directory_sync.py

```diff
@@ -34,13 +34,13 @@ def request_helper(self):
 
     def list_users(
         self,
-        directory=None,
-        group=None,
-        limit=None,
-        before=None,
-        after=None,
-        order=None,
-    ):
+        directory: Optional[str] = None,
+        group: Optional[str] = None,
+        limit: int = RESPONSE_LIMIT,
+        before: Optional[str] = None,
+        after: Optional[str] = None,
+        order: PaginationOrder = "desc",
```

Comment:
`Optional[PaginationOrder] = "desc"` is not what we want here. That allows `None` as a possible value. This disallows `None` and does not require `order` to be specified.